src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {Fragment} from 'react';
+import {StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+import Text from '@components/Text';
+import useThemeStyles from '@styles/useThemeStyles';
+import CONST from '@src/CONST';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+
+type WrappedTextProps = ChildrenProps & {
+    /** Style to be applied to Text */
+    textStyles?: StyleProp<TextStyle>;
+
+    /** Style for each word(Token) in the text, remember that token also includes whitespaces among words */
+    wordStyles?: StyleProp<ViewStyle>;
+};
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,' ',Name,' ',' ',is,' ',Rajat],
+ *  ]
+ */
+function getTextMatrix(text: string): string[][] {
+    return text.split('\n').map((row) => row.split(CONST.REGEX.SPACE_OR_EMOJI).filter((value) => value !== ''));
+}
+
+function WrappedText({children, wordStyles, textStyles}: WrappedTextProps) {
+    const styles = useThemeStyles();
+
+    if (typeof children !== 'string') {
+        return null;
+    }
+
+    const textMatrix = getTextMatrix(children);
+
+    return textMatrix.map((rowText, rowIndex) => (
+        <Fragment
+            // eslint-disable-next-line react/no-array-index-key
+            key={`${rowText[0]}-${rowIndex}`}
```

Comment:
Does this change to from `rowText` to `rowText[0]` have any effect?

Reply:
Since `rowText` is an array, it's not good to use it inside key. Also, it has a warning
![image](https://github.com/Expensify/App/assets/23176449/cdd74353-80b0-41c6-96f2-cced36c46052)